### PR TITLE
stopwatch notif: check if timetracking is enabled

### LINKED ISSUE #14435: stopwatches endpoint should not be triggered when ENABLE_TIMETRACKING = false
- Gitea version: 1.14

## Description

The /api/v1/user/stopwatches endpoint is queried every 10 seconds, even when `ENABLE_TIMETRACKING = false` in app.ini

```
2021/01/22 20:38:03 Started GET /api/v1/user/stopwatches for 127.0.0.1:52339
2021/01/22 20:38:03 Completed GET /api/v1/user/stopwatches 200 OK in 9.9781ms
2021/01/22 20:38:13 Started GET /api/v1/user/stopwatches for 127.0.0.1:52339
2021/01/22 20:38:13 Completed GET /api/v1/user/stopwatches 200 OK in 11.0478ms
2021/01/22 20:38:23 Started GET /api/v1/user/stopwatches for 127.0.0.1:52339
2021/01/22 20:38:24 Completed GET /api/v1/user/stopwatches 200 OK in 8.9847ms
2021/01/22 20:38:34 Started GET /api/v1/user/stopwatches for 127.0.0.1:52339
2021/01/22 20:38:34 Completed GET /api/v1/user/stopwatches 200 OK in 10.9856ms
2021/01/22 20:38:44 Started GET /api/v1/user/stopwatches for 127.0.0.1:52339
2021/01/22 20:38:44 Completed GET /api/v1/user/stopwatches 200 OK in 10.0009ms
2021/01/22 20:38:54 Started GET /api/v1/user/stopwatches for 127.0.0.1:52339
2021/01/22 20:38:54 Completed GET /api/v1/user/stopwatches 200 OK in 9.9988ms
2021/01/22 20:39:04 Started GET /api/v1/user/stopwatches for 127.0.0.1:52339
2021/01/22 20:39:04 Completed GET /api/v1/user/stopwatches 200 OK in 10.0026ms
2021/01/22 20:39:14 Started GET /api/v1/user/stopwatches for 127.0.0.1:52339
2021/01/22 20:39:14 Completed GET /api/v1/user/stopwatches 200 OK in 10.9982ms
```

Relates to #14122

### PR DESCRIPTION
**Pull request by noerw**
fixes #14435

### CODE REVIEW

File: templates/base/head.tmpl [hunk 1]
```diff
@@ -42,6 +42,7 @@
 				MaxTimeout: {{NotificationSettings.MaxTimeout}},
 				EventSourceUpdateTime: {{NotificationSettings.EventSourceUpdateTime}},
 			},
+			EnableTimetracking: {{if EnableTimetracking}}true{{else}}false{{end}},
```
Comment (techknowlogick) [inline templates/base/head.tmpl:45]: as a side note, we could probably get these into the system info api endpoint to better integrate SPAs in the future (this isn't a note for this PR though).

Comment (6543) [reply templates/base/head.tmpl:45]: same with lfs enable/disable info ... - will add a pull

**Review by 6543 — APPROVED** [review]

File: templates/base/head.tmpl (hunk 1 above)
Comment (6543) [reply templates/base/head.tmpl:45]: https://github.com/go-gitea/gitea/pull/14753

**Review by zeripath — APPROVED** [review]

**Comment by 6543** [comment]
:rocket:

File: templates/base/head.tmpl (hunk 1 above)
Comment (techknowlogick) [reply templates/base/head.tmpl:45]: awesome. thanks :)

**Pull request merged**